### PR TITLE
Fixing IpFabric to work on WS2019

### PR DESCRIPTION
**Pull request by jacekchalupka**

### CODE REVIEW

File: Test/Tests/Network/IPFabric.Tests.ps1 [hunk 1]
```diff
@@ -88,17 +127,17 @@ Test-WithRetries 3 {
...
+            $BeforeEachStack.Push(${function:Remove-AllContainers}, @(, $Testenv.Testbeds))
 
             Write-Log "Creating container: $ContainerID"
             New-Container `
-                -Testbed $Testenv.Sessions[0] `
+                -Testbed $Testenv.Testbeds[0] `
                 -NetworkName $VirtualNetwork.Name `
                 -Name $ContainerID `
-                -Image $DockerImage
+                -Image $Testbed.DefaultDockerImage
```
Comment (michal-clapinski) [inline Test/Tests/Network/IPFabric.Tests.ps1]: can't you just not pass this param?

File: Test/Tests/Network/IPFabric.Tests.ps1 [hunk 2]
```diff
@@ -25,23 +25,60 @@ Param (
...
+
+    $NetPrefixMask = [uInt32]::MaxValue -shl (32 - $IpInfo.PrefixLength)
+    $NetIp = Convert-IpStringToUInt32 -Ip $IpInfo.IPAddress
+    $NetPrefix = $NetIp -band $NetPrefixMask
+    $NetBroadcast = $NetPrefix + (-bnot $NetPrefixMask)
+
+    $SubnetPoolEnd = $NetBroadcast - 1
+    $SubnetPoolBeg = $NetBroadcast - 1
+    $SubnetDefaultGate = $NetBroadcast - 6
+    $SubnetPrefix = $NetBroadcast - 7
```
Comment (michal-clapinski) [inline Test/Tests/Network/IPFabric.Tests.ps1]: please add some comments here, especially why the new PrefixLength is 29

File: Test/Tests/Network/IPFabric.Tests.ps1 (hunk 1 above)
Comment (jacekchalupka) [reply Test/Tests/Network/IPFabric.Tests.ps1]: I can, forgot to change when introduced this option.

File: Test/Tests/Network/IPFabric.Tests.ps1 (hunk 2 above)
Comment (jacekchalupka) [reply Test/Tests/Network/IPFabric.Tests.ps1]: Done

**Comment by jacekchalupka** [comment]
recheck no bug

**Pull request merged**